### ISSUE TITLE
Option separators in <switches>

### ISSUE THREAD

**Issue by gyurco** [jotego/jtbin#284]
In this patch:
06a6b41e51daca6f69077f672d9f1f44754164ee
How do you make a difference between ';' and ',' as an option separator?
E.g.
Free Play or 1/1,1/1;
should be handled as a whole, but at the end:
6/4,2/3,4/1,3/1,2/1,1/6,1/5,1/4,1/3,1/2,1/1
these are 11 separate options

**Comment by MJY71** [comment]
As I outlined in [this](https://github.com/jotego/jtbin/issues/279) issue, MiSTer treats commas as DIP switch permutation separators and therefore cannot be used in the DIP switch descriptions for the OSD, unlike MAME.

What Jotego ended up doing was slightly different than what I suggested. The semicolons are just for display purposes in the OSD. They are not functional with regards to separating one valid DIP switch permutation from the next.

There should be 16 total valid options to cycle through for both Coin A and B.
